### PR TITLE
RELATED: ONE-3336 ONE-3339 Fix gridline and cut-off points in bubble chart

### PR DESCRIPTION
**Pull request by peterlacko**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1408/ : SUCCESS in 1m 00s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/902/ : FAILURE in 7m 46s

**Comment by peterlacko** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/511/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1414/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/266/ : UNSTABLE in 13m 47s

File: src/components/visualizations/chart/highcharts/customConfiguration.ts [hunk 1]
```diff
@@ -325,12 +325,31 @@ function level2LabelsFormatter(config?: IChartConfig) {
     return `${get(this, 'point.name')} (${formatLabel(get(this, 'point.value'), get(this, 'point.format'), config)})`;
 }
 
+function isNullOrUndefined(value: any) {
```
Comment (pbenes) [inline src/components/visualizations/chart/highcharts/customConfiguration.ts]: maybe `_.isNil(value)` from lodash does the same? please check

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1422/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/914/ : SUCCESS in 3m 07s

File: src/components/visualizations/chart/highcharts/customConfiguration.ts (hunk 1 above)
Comment (peterlacko) [reply src/components/visualizations/chart/highcharts/customConfiguration.ts]: 👍 fixed

**Comment by peterlacko** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1451/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/950/ : SUCCESS in 2m 05s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/550/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1452/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/283/ : SUCCESS in 12m 58s

**Comment by gdgate** [comment]
Build failed (check-teardown pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-stop-zuul-docker/144/ : FAILURE in 3m 39s

**Pull request closed without merging**